build.gradle

```diff
@@ -263,6 +263,7 @@ dependencies {
     compile 'org.fusesource.jansi:jansi:1.11'
     compile 'org.yaml:snakeyaml:1.17'
     compile 'com.google.code.gson:gson:2.6.2'
+    compile 'com.github.spullara.mustache.java:compiler:0.9.1'
```

Comment:
Licence needs to be added to the README.  It also needs to be added to the distribution.